docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -20,6 +20,7 @@ The following APIs support {ccs}:
 * <<multi-search-template,Multi search template>>
 * <<search-field-caps,Field capabilities>>
 * experimental:[] <<eql-search-api,EQL search>>
+* experimental:[] <<sql-search-api,SQL search>>
```

Comment:
Is this experimental for SQL? If so, we should include this `experimental:[]` elsewhere in the docs.

Reply:
@jrodewig I've added the `experimental:[]` tag elsewhere too, let me know please if this is OK. Thanks!

Reply:
I double-checked these, and the tags look good overall. I did leave some suggestions related to `cross cluster search` references, which should be hyphenated.

Reply:
Thanks, @jrodewig!